src/callbacks/callbacks.jl

```diff
@@ -128,9 +128,7 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
         end
     end
 
-    if !idealized_insolation && !(p.atmos.surface_albedo isa CouplerAlbedo)
-        set_insolation_variables!(Y, p, t)
-    end
+    set_insolation_variables!(Y, p, t, p.atmos.insolation)
```

Comment:
```suggestion
    if p.atmos.insolation isa IdealizedInsolation || !(p.atmos.surface_albedo isa CouplerAlbedo)
        set_insolation_variables!(Y, p, t, p.atmos.insolation)
    end
```
(I think this is closer to our previous logic, and we can remove the check of albedo in the actual function, which may cause some issues for ClimaCoupler.)